src/modules/dashboard/components/CoinMachine/SynapsKYCDialog/SynapsKYCDialog.css

```diff
@@ -5,6 +5,10 @@
   height: 730px;
 }
 
+.content > div > div {
+  margin-bottom: 30px;
```

Comment:
This may be a little too broad targeting. Especially if different forms of KYC is used in future which we don't know the structure of. Perhaps we could add a class to wrap around the loader so that it can be targeted specifically.

Reply:
Thanks you. It's a good point, i like your suggestion. I will refactor.